src/controllers/user.js

```diff
@@ -95,9 +97,14 @@ const User = {
   fetch: async (req, res, next) => {
     const { id } = req.params;
     let result;
+    let imgURL;
 
     try {
       result = await UserModel.findOne({ _id: id }).exec();
+      imgURL = await S3.getPreSignedURL({
+        Bucket: 'image-sharing-project',
+        Key: `users/${result.id}/profile.jpeg`,
+      });
     } catch (err) {
       return next(new APIError());
```

Comment:
Same here, error code would be good, but it might already give it somewhere else